### PR TITLE
ci: remove gpg passphrase

### PR DESCRIPTION
**Pull request by blaggacao**
# Context

The new Lace Bot GPG key has no passphase.

# Proposed Solution

Remove the passphrase config variable.

# Testing

- [x] run the ci

### CODE REVIEW

**Comment by mkazlauskas** [comment]
Please test the workflow change by running `release` with workflow_dispatch against this branch workflow.

**Comment by blaggacao** [comment]
> Please test the workflow change by running release with workflow_dispatch against this branch workflow.

&rarr; https://github.com/input-output-hk/cardano-js-sdk/actions/runs/6158389380 :pray:

**Review by mkazlauskas — APPROVED** [review]
Nice work!

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -7,48 +7,47 @@ jobs:
...
+        with:
+          node-version: 18.12.0
+          registry-url: 'https://registry.npmjs.org'
 
-    - name: Import bot's GPG key for signing commits
-      id: import-gpg
-      uses: crazy-max/ghaction-import-gpg@v5
-      with:
-        gpg_private_key: ${{ secrets.BOT_GPG_KEY }}
-        passphrase: ${{ secrets.BOT_GPG_PASSPHRASE }}
```
Comment (gytis-ivaskevicius) [inline .github/workflows/release.yaml:32]: I guess this is the only relevant change? It feels weird to have gpg key without a passphrase but I guess it does not make much difference

**Review by gytis-ivaskevicius — APPROVED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (blaggacao) [reply .github/workflows/release.yaml:32]: Both secrets are held by the GH action and the same GH secret store, so a passphrase only adds complexity at no gain in security.

**Pull request merged**